ports.js

```diff
@@ -1 +1,57 @@
-app = Elm.Main.fullscreen();
+const KINTO_SERVER = "https://kinto.dev.mozaws.net/v1";  // No trailing slash
+const KINTO_BUCKET = "stepfunction";
+const KINTO_COLLECTION = "manual_steps";
```

Comment:
Can we get rid of those three, and have them in the elm file, so we minimize what we deal with in js? I think it's only used in the `form.action` below, which could be managed in elm?